LeanPoly/Poly.lean

```diff
@@ -64,6 +65,7 @@ structure Poly where
   dir : pos -> Type v
 
 /-- The type of lenses/maps from one polynomial functor to another. -/
+@[ext]
```

Comment:
Handy macro for proving equality of lenses: makes it so that by calling the `ext` tactic in such a goal, splits it into two goals, one for the position and one for the direction map